packages/ui/__stories__/NumberField.stories.tsx

```diff
@@ -33,7 +33,9 @@ const Template: Story<NumberFieldProps> = ({ value: initValue, ...args }) => {
   return (
     <>
       <NumberField {...args} value={value} onChange={setValue} />
+      <NumberField {...args} value={value} onChange={setValue} iconPosition="together" className={storyStyles.shortField} />
```

Comment:
@allfayn @pawelklasa Should we do something with placeholder?
![image](https://user-images.githubusercontent.com/13898415/139834631-d0368af1-4c81-42d0-ae63-7f13a7a2793d.png)

Reply:
removed placeholders for this kind of fields at story